src/app/@auth/[...catchAll]/page.tsx

```diff
@@ -0,0 +1,3 @@
+export default function CatchAll() {
+  return null;
+}
```

Comment:
This [...catchAll] directory and null return is required according to the Next docs. It ensures that the modal is closed when navigating to a new route. 
Before adding this, the register modal would still be open when I navigated to the terms of service link